src/gmusic-theme.js

```diff
@@ -11,7 +11,7 @@ const CONSTANTS = require('../lib/_constants');
 const DEFAULTS = {
   BACK_PRIMARY: '#222326',
   BACK_SECONDARY: '#121314',
-  BACK_HIGHLIGHT: '#615F59',
+  BACK_HIGHLIGHT: '#1a1b1d',
```

Comment:
Let's leave this in GPMDP only for now.  Making this change and following semver (like I do) would require this to be a breaking major version jump.  And I hate those